src/linagora.esn.profile/app/app.module.js

```diff
@@ -32,7 +33,8 @@ require('esn-frontend-common-libs/src/frontend/js/modules/user-configuration/use
     'esn.async-action',
     'esn.availability',
     'esn.module-registry',
-    'esn.user-configuration'
+    'esn.user-configuration',
+    'esn-frontend-account-inbox'
```

Comment:
inject the Angular module instead of SPA name
```suggestion
    'esn.account-inbox'
```